deployment.md

```diff
@@ -0,0 +1,33 @@
+# VFOX 0.1 Deployment Guide
+
+### Purpose
+ Contain information about the installation process of backend as JAVA and frontend as Angular.
+### Prerequisites
+```
+Java 1.8+
+Mysql 5.x
+docker 1.3x
```

Comment:
Why do we need 'docker' on the instance?

Reply:
Removed 'docker' as it is already been taken care in README.md file.
Earlier, it was included just as a Prerequisite.